src/Mutator/Util/MutatorProfile.php

```diff
@@ -191,6 +191,7 @@
         Mutator\Unwrap\UnwrapArrayFlip::class,
         Mutator\Unwrap\UnwrapArrayMap::class,
         Mutator\Unwrap\UnwrapArrayReverse::class,
+        Mutator\Unwrap\UnwrapStrtolower::class,
```

Comment:
Could you please name it `StrToLower` to be in line with https://github.com/infection/infection/pull/535?
Documentation should also be updated in this case

Reply:
@borNfreee 

Done!